### DISCUSSION TITLE
post/random-virus-reverse/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#8] [Announcements]
# post/random-virus-reverse/

0x00 前言最近高校迎来分手毕业季, 大批大四同学们的毕业论文都要去校内的打印店打印装订. 于是毫无意外的, 学校打印店无人管理的公共电脑成功地被 U 盘病毒感染了. (看来即使是在 985 高校, 你也永远不应该相信大学生的电脑使用水平) 不少同学用 U 盘拷贝文件的时候就中招了, 看群友的反馈描述, 这次传播的不是最经典的那种伪装成文件夹的 exe 病毒. 3202 年的我已经很久没有在现实

https://blog.lyc8503.site/post/random-virus-reverse/

**Comment by lyc8503** [comment]
在 Avira 上也找到同样样本的一份 write-up https://www.avira.com/en/blog/new-wave-of-plugx-targets-hong-kong, 提及了一些本文没提到的技术细节, 同样供参考.

**Comment by mhshan925** [comment]
链接访问不到了，可以发邮箱吗？15073075360@163.com
